src/frontend/apps/e2e/__tests__/app-impress/doc-header.spec.ts

```diff
@@ -125,7 +125,7 @@ test.describe('Doc Header', () => {
       .fill(' ');
 
     await page.getByText('Created at').click({
-      delay: 200,
+      delay: 1000,
     });
```

Comment:
remove this, by default we have 5.000 ms

Reply:
I removed this delay.
I can clearly reproduce the flakiness, I didn't find better than adding that:
https://github.com/numerique-gouv/impress/pull/511/commits/19224026f02e622420c93213b253c28fcf95607d#diff-8c5f9b668095c002522e065079a61f46ea012038e2eec0b6d88e8b2411d55e6dR124

Lot of states are going on very quickly, it needs a delay to stabilize.